neuralmonkey/writers/plain_text_writer.py~

```diff
@@ -0,0 +1,31 @@
+from typing import TypeVar, Iterator, List, Any, Callable
```

Comment:
Again, the .py~ file should probably not be included in the commit.

Reply:
dkuji